frontend/src/data/variables/BrfssProvider.ts

```diff
@@ -29,7 +39,48 @@ class BrfssProvider extends VariableProvider {
     df = this.filterByGeo(df, breakdowns);
     df = this.renameGeoColumns(df, breakdowns);
 
+    // TODO How to handle territories?
+    let acsBreakdowns = breakdowns.copy();
+    acsBreakdowns.time = false;
+
+    let consumedDatasetIds = ["brfss"];
+
     if (breakdowns.geography === "national") {
+      // Because we add together the estimated state totals
+      // we need to get the acs state breakdown
+      acsBreakdowns.geography = "state";
+
+      const acsQueryResponse = await this.acsProvider.getData(
+        new MetricQuery(["population", "population_pct"], acsBreakdowns)
```

Comment:
Do you actually use population_pct from the ACS provider anywhere? If not, you can just not request this metric

Reply:
Yeah I think I can get rid of it, thanks